pkg/manifests/manifests_test.go

```diff
@@ -1573,6 +1574,136 @@ alertmanagerMain:
 	}
 }
 
+func TestK8sPrometheusAdapterAuditLog(t *testing.T) {
+	// setting audit level should add the audit args
+	tt := []struct {
+		config       string
+		expectedArgs []string
+		err          error
+	}{{
+		config: ``,
+		expectedArgs: []string{
+			"--audit-policy-file=/etc/audit/default-profile.yaml",
+			"--audit-log-path=/var/log/adapter/audit.log",
+			"--audit-log-maxsize=100",
```

Comment:
I see there is `--audit-log-maxage` paramater for prometheus adapter, is there a default value for it?
```--audit-log-maxage int                                    The maximum number of days to retain old audit log files based on the timestamp encoded in their filename.```

Reply:
yes, I think `--audit-log-maxsize` (which is also optional) should be sufficient.